canonicalwebteam/django_views/__init__.py

```diff
@@ -51,7 +51,9 @@ def _find_matching_template(path):
     """
 
     # Try to match HTML or Markdown files
-    if _template_exists(path + ".html"):
+    if _template_exists(path):
```

Comment:
Shouldn't `/index.html` redirect to `/`? And `/mypage.html` redirect to `/mypage`? I don't think we should have 2 paths lead to the same content.

Reply:
E.g. (roughly):

```suggestion
    if _template_exists(path) and path.endswith('/index.html'):
        return flask.redirect(path[:-11])
    elif  _template_exists(path) and path.endswith('/index'):
        return flask.redirect(path[:-6])
    elif  _template_exists(path) and path.endswith('.html'):
        return flask.redirect(path[:-5])
```